src/generators/php/index.js

```diff
@@ -141,11 +156,22 @@ class PHP extends Generator {
       case "null":
         return "null";
       default:
-        if (enumMap[typeName]) {
-          // TODO: OA and SchemaOrg use same namespace: do we need to differentiate?
+        if (enumMap[compactedTypeName]) {
+          if (this.includedInSchema(enumMap[compactedTypeName].namespace)) {
+            return (
+              "\\OpenActive\\Enums\\SchemaOrg" +
```

Comment:
Missing trailing `\\` so that fully namespaced name is for example `\OpenActive\Enums\SchemaOrg\EntryPoint` instead of `\OpenActive\Models\SchemaOrgEntryPoint`

Reply:
Good spot